qt/res/app.organicmaps.desktop.metainfo.xml

```diff
@@ -107,6 +107,22 @@
   </screenshots>
 
   <releases>
+   <release version="2022.07.27-3" date="2022-07-27">
+     <description>
+       <ul>
+         <li>Fixed invalid opertions on Wayland for OpenGL ES3 capable devices</li>
+         <li>Use OpenGL ES3 by default on Linux</li>
+         <li>New OSM data as of July 18</li>
+         <li>Fixed/updated API (see omaps.app/api)</li>
+         <li>Added missing shop types from top OSM tags</li>
+         <li>Minor routing improvement</li>
+         <li>Use min_level for 3D buildings</li>
+         <li>Allow removing cuisines from selection in the Editor</li>
+         <li>Fixed be, de, ru, uk translations</li>
+         <li>Fixed FB URLs validation</li>
```

Comment:
The last 3 entries are not relevant to the desktop version.

See https://github.com/flathub/app.organicmaps.desktop/pull/13#pullrequestreview-1054857409

Reply:
OK the last 3 entries have been removed.